shared/hybrid/AppDelegate+SalesforceHybridSDK.m

```diff
@@ -49,7 +49,7 @@ + (void)load
 - (AppDelegate *)sfsdk_swizzled_init
 {
     // Need to use SalesforceSDKManagerWithSmartStore when using smartstore
-    [SalesforceSDKManager setInstanceClass:[SalesforceSDKManagerWithSmartStore class]];
+    [SalesforceSDKManager setInstanceClass:[SalesforceHybridSDKManager class]];
```

Comment:
I need to update the native / react native template also.
I will do that once this PR goes in.